docs/index.md

```diff
@@ -107,14 +114,18 @@ We then define a set of services using `docker-compose.yml`:
 
 This defines two services:
 
- - `web`, which is built from `Dockerfile` in the current directory. It also says
-   to run the command `python app.py` inside the image, forward the exposed port
-   5000 on the container to port 5000 on the host machine, connect up the Redis
-   service, and mount the current directory inside the container so we can work
-   on code without having to rebuild the image.
- - `redis`, which uses the public image [redis](https://registry.hub.docker.com/_/redis/). 
+ - `web`, which is built from the `Dockerfile` in the current directory. It also
+ says to run the command `python app.py` inside the image, forward the exposed
```

Comment:
I think these need to be indented as before, otherwise they won't be a part of the bullet point.